src/Bathymetry.jl

```diff
@@ -115,10 +115,10 @@ function regrid_bathymetry(target_grid;
     λ_data = dataset["lon"][:]
     z_data = convert(Array{FT}, dataset["z"][:, :])
 
-    # Convert longitude to 0 - 360?
+    # Convert longitude from (-180, 180) to (0, 360)
     λ_data .+= 180
-    nhx      = size(z_data, 1)
-    z_data   = circshift(z_data, (nhx ÷ 2, 1))
+    Nhx    = size(z_data, 1)
+    z_data = circshift(z_data, (Nhx ÷ 2, 1))
```

Comment:
I think this should be 
```suggestion
    z_data = circshift(z_data, (Nhx ÷ 2, 0))
```
because the second argument of the circshift are the offsets